src/python_testing/TC_ACL_2_7.py

```diff
@@ -0,0 +1,293 @@
+from mobly import asserts
+
+D_OK_EMPTY = bytes.fromhex('1718')
+D_OK_SINGLE = bytes.fromhex(
+    '17D00000F1FF01003D48656C6C6F20576F726C642E205468697320697320612073696E676C6520656C656D656E74206C6976696E6720617320612063686172737472696E670018')
+
+
+class TC_ACL_2_7(MatterBaseTest):
+    def desc_TC_ACL_2_7(self) -> str:
+        return "[TC-ACL-2.7] Multiple fabrics test"
+
+    def steps_TC_ACL_2_7(self) -> list[TestStep]:
+        steps = [
+            TestStep(1, "TH1 commissions DUT using admin node ID",
+                     is_commissioning=True),
+            TestStep(2, "TH1 reads DUT Endpoint 0 OperationalCredentials cluster CurrentFabricIndex attribute",
+                     "Result is SUCCESS, value is stored as F1"),
+            TestStep(3, "TH1 puts DUT into commissioning mode, TH2 commissions DUT using admin node ID",
+                     "DUT is commissioned on TH2 fabric"),
+            TestStep(4, "TH2 reads DUT Endpoint 0 OperationalCredentials cluster CurrentFabricIndex attribute",
+                     "Result is SUCCESS, value is stored as F2"),
+            TestStep(5, "TH1 writes DUT Endpoint 0 AccessControl cluster Extension attribute, value is list of AccessControlExtensionStruct containing 1 element", "Result is SUCCESS"),
+            TestStep(6, "TH2 writes DUT Endpoint 0 AccessControl cluster Extension attribute, value is list of AccessControlExtensionStruct containing 1 element", "Result is SUCCESS"),
+            TestStep(7, "TH1 reads DUT Endpoint 0 AccessControl cluster Extension attribute",
+                     "Result is SUCCESS, value is list of AccessControlExtensionStruct containing 1 element"),
+            TestStep(8, "TH2 reads DUT Endpoint 0 AccessControl cluster Extension attribute",
+                     "Result is SUCCESS, value is list of AccessControlExtensionStruct containing 1 element"),
+            TestStep(9, "TH1 reads DUT Endpoint 0 AccessControl cluster AccessControlExtensionChanged event",
+                     "Result is SUCCESS, value is list of AccessControlExtensionChanged containing 1 element"),
+            TestStep(10, "TH1 reads DUT Endpoint 0 AccessControl cluster AccessControlExtensionChanged event",
+                     "Result is SUCCESS, value is list of AccessControlExtensionChanged containing 1 element"),
+        ]
+        return steps
+
+    @async_test_body
+    async def test_TC_ACL_2_7(self):
+        self.step(1)
+        self.th1 = self.default_controller
+        self.discriminator = random.randint(0, 4095)
+
+        self.step(2)
+        # Read CurrentFabricIndex for TH1
+        oc_cluster = Clusters.OperationalCredentials
+        cfi_attribute = oc_cluster.Attributes.CurrentFabricIndex
+        result = await self.th1.ReadAttribute(
+            self.dut_node_id,
+            [(0, cfi_attribute)]
+        )
+        f1 = result[0][oc_cluster][cfi_attribute]
+        self.print_step("CurrentFabricIndex F1", str(f1))
```

Comment:
can you just logging for these? the print_step function adds some stars etc to make it look like test step